scripts/combined_commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I think this file is good as is. But if you want, you could try running `shellcheck` tool over this file, like `shellcheck combined_commands.sh`. `shellcheck` is a linter for bash and it will spot most bash mistakes.